overlord/snapstate/handlers.go

```diff
@@ -838,7 +838,41 @@ func (m *SnapManager) doDiscardSnap(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
-// aliases v2
+/* aliases v2
+
+aliases v2 implementation uses the following tasks:
+
+  * for install/refresh/remove/enable/disable etc
+
+    - remove-aliases: remove aliases of a snap from disk and mark them pending
+
+    - setup-aliases: (re)creates aliases from snap state, mark them as
+      not pending
+
+    - set-auto-aliases: updates aliases snap state based on the
+      snap-declaration and current revision info of the snap
```

Comment:
refresh-aliases is  equivalent (almost) to  set-auto-aliases+setup-aliases, except it doesn't change pending state

set-auto-aliases existed already in the old world with similar functionality, given that now for example core update is a two step process, if we try to change it around we need to do surgery or kill pending install changes across the update, but then we need to special case at least core. we can rethink this later once we assume the transition has gone through or we have epochs but for now it seems safer to keep this structure